spec/models/manageiq/providers/hawkular/middleware_manager/middleware_server_spec.rb

```diff
@@ -13,20 +13,48 @@
   let(:eap) do
     FactoryGirl.create(:hawkular_middleware_server,
                        :name                  => 'Local',
-                       :feed                  => '377fadac-98eb-4a53-bf3b-89c21bccd8ba',
+                       :feed                  => '27e06e8f-6e5d-45cd-8353-c1f9d4dfe991',
                        :ems_ref               => '/t;hawkular'\
-                                                 '/f;377fadac-98eb-4a53-bf3b-89c21bccd8ba/r;Local~~',
+                                                 '/f;27e06e8f-6e5d-45cd-8353-c1f9d4dfe991/r;Local~~',
                        :nativeid              => 'Local~~',
                        :ext_management_system => ems_hawkular)
   end
 
+  let(:expected_metrics) do
+    {
+      "WildFly Memory Metrics~Heap Used"                                  => "mw_heap_used",
+      "WildFly Memory Metrics~Heap Max"                                   => "mw_heap_max",
+      "WildFly Memory Metrics~Heap Committed"                             => "mw_heap_committed",
+      "WildFly Memory Metrics~NonHeap Used"                               => "mw_non_heap_used",
+      "WildFly Memory Metrics~NonHeap Committed"                          => "mw_non_heap_committed",
+      "WildFly Memory Metrics~Accumulated GC Duration"                    => "mw_accumulated_gc_duration",
+      "WildFly Aggregated Web Metrics~Aggregated Servlet Request Time"    => "mw_agregated_servlet_time",
+      "WildFly Aggregated Web Metrics~Aggregated Servlet Request Count"   => "mw_aggregated_servlet_request_count",
+      "WildFly Aggregated Web Metrics~Aggregated Expired Web Sessions"    => "mw_aggregated_expired_web_sessions",
+      "WildFly Aggregated Web Metrics~Aggregated Max Active Web Sessions" => "mw_aggregated_max_active_web_sessions",
+      "WildFly Aggregated Web Metrics~Aggregated Active Web Sessions"     => "mw_aggregated_active_web_sessions",
+      "WildFly Aggregated Web Metrics~Aggregated Rejected Web Sessions"   => "mw_aggregated_rejected_web_sessions",
+      "WildFly Threading Metrics~Thread Count"                            => "mw_thread_count",
+      "Server Availability~App Server"                                    => "mw_availability_app_server",
+      "Transactions Metrics~Number of Aborted Transactions"               => "mw_tx_aborted",
+      "Transactions Metrics~Number of In-Flight Transactions"             => "mw_tx_inflight",
+      "Transactions Metrics~Number of Committed Transactions"             => "mw_tx_committed",
+      "Transactions Metrics~Number of Transactions"                       => "mw_tx_total",
+      "Transactions Metrics~Number of Application Rollbacks"              => "mw_tx_application_rollbacks",
```

Comment:
"The number of transactions that have been rolled back by application request. This includes those that timeout, since the timeout behaviour is considered an attribute of the application configuration."
https://access.redhat.com/documentation/en-US/JBoss_Operations_Network/3.1/html/Dev_Complete_Resource_Reference/JBossAS7-JBossAS7_Standalone_Server-Transactions_Subsystem_-Standalone-.html